README.md

```
@@ -0,0 +1,157 @@
+# horizons
+
+> Spectral Ensemble Modeling for Soil Property Prediction
+
+<img src="man/figures/logo.png" align="right" width="140"/>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)